pkg/context/mqttcontext.go

```diff
@@ -88,8 +93,11 @@ const (
 	// MQTTPublish is mqtt packet type of publish
 	MQTTPublish MQTTPacketType = 2
 
+	// MQTTDisconnect is mqtt packet type of disconnect
+	MQTTDisconnect = 3
+
 	// MQTTOther is all other mqtt packet type
-	MQTTOther MQTTPacketType = 3
+	MQTTOther MQTTPacketType = 4
```

Comment:
Let MQTTOther's value equal a large value, for example, `999` or `9999`, to avoid value conflict in the future.

Reply:
Good idea!